.gitignore

```diff
@@ -0,0 +1,5 @@
+
+bazel-bin
```

Comment:
These are top-level directories, right? They should be prefixed with a slash:

    /bazel-bin

Without the slash this will ignore `bazel-bin` files everywhere in the tree.